internal/archive/tar/tar.go

```diff
@@ -0,0 +1,70 @@
+package tar
+
+import (
+	"archive/tar"
+	"bytes"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/saucelabs/saucectl/internal/sauceignore"
+)
+
+// Resource archives the resource and exclude files and folders based on sauceignore logic.
+func Resource(src string, matcher sauceignore.Matcher) (io.Reader, error) {
```

Comment:
How about?
```suggestion
func Archive(src string, matcher sauceignore.Matcher) (io.Reader, error) {
```

or
```suggestion
func Create(src string, matcher sauceignore.Matcher) (io.Reader, error) {
```

Feel like `tar.Create()` and `tar.Archive` reads itself a bit more intuitively to me? :D

Reply:
same idea :D